examples/Demo/Shared/Pages/Grid/GridPage.razor

```diff
@@ -54,6 +54,12 @@
     To display an element only on a given range of screen sizes, you can combine a these values.
     e.g. <code>GridItemHidden.Md | GridItemHidden.Lg</code> will display the element for all screen sizes, except on medium and large devices.
 </p>
+<p>
+    <b>AdaptiveRendering</b> allows you not to generate the HTML code of the <b>FluentGridItem</b> that respects the <code>HiddenWhen</code>.
```

Comment:
<b>AdaptiveRendering</b> allows for not generating HTML code in a  <b>FluentGridItem</b> based on the value of the <code>HiddenWhen</code> parameter.